src/components/pie-stimulus-layout/pie-stimulus-layout.tsx

```diff
@@ -39,6 +43,21 @@ export class PieStimulusLayout {
       window.addEventListener('mousemove', this.handleMouseMove.bind(this));
       this.resizer.addEventListener('mouseup', this.handleMouseUp.bind(this));
     }
+
+    this.readMoreButton.addEventListener("click", () => {
```

Comment:
Can you please create a function for the event handler instead of having everything here?